crates/weaver_forge/src/debug.rs

```diff
@@ -2,14 +2,34 @@
 
 //! Utility functions to help with debugging.
 
-use crate::error::Error;
 use crate::error::Error::{CompoundError, TemplateEvaluationFailed};
 use indexmap::IndexMap;
+use std::error::Error;
 use weaver_common::Logger;
 
-/// Return a nice summary of the error.
+/// Return a nice summary of the error including the chain of causes.
+/// Only the last error in the chain is displayed with a full stack trace.
 pub(crate) fn error_summary(error: minijinja::Error) -> String {
```

Comment:
If this does what I think it does, maybe it's time to move the Docker image from debug=>release?

I can start a tracker but it'd be nice to rely on release builds (~100MB vs. 400MB somehow).

Reply:
I don't remember the exact problem you observed with the release version.

Reply:
So there were two issues:

1. The error output was truncated to the point of being useless, whereas in debug build you got line numbers and pointers at problematic templates.
2. The `{{ debug(var) }}` trick for templates stopped rendering.

Reply:
I will check that